pkg/scheduler/framework/interface.go

```diff
@@ -229,15 +231,15 @@ type Plugin interface {
 }
 
 // LessFunc is the function to sort pod info
-type LessFunc func(podInfo1, podInfo2 *QueuedPodInfo) bool
+type LessFunc func(podInfo1, podInfo2 *v1alpha1.QueuedPodInfo) bool
 
```

Comment:
this should be *framework

Reply:
anything in the scheduler pkg should point to the framework pkg

Reply:
fixed